api/gas/index.js

```diff
@@ -33,23 +31,8 @@ module.exports = (stores, useCases) => {
       })
   });
 
-  app.put('/gasStations/:gasStationId', function(req, res) {
-    // Customer -> Block
-    // Dispatcher -> Enable/Disable own gas station
-    const useCase = new useCases.HostessToggleGasStation({
-      branchId: req.params.gasStationId,
-      hostessId: req.body.hostess_id,
-      hostessStore: stores.hostessStore,
-      branchStore: stores.branchStore,
-    });
-
-    useCase.execute()
-      .then(_ => res.json(_))
-      .catch(error => res.json(error))
-  });
-
+  /*
```

Comment:
Delete commented code 🐷

Reply:
Just need to figure out how to handle private and public logic in a same endpoint. Do you know how to do that?

Reply:
Maybe implement a wrapper around your current auth and pass a parameter to tell if authentication should be enforced or not. But that is a problem for another time